inductiva/_cli/main.py

```diff
@@ -42,6 +44,25 @@ def get_main_parser():
     return parser
 
 
+def watch(func, every, args, cmd):
+    """Run the function at regular intervals and display results in a pager."""
+    cmd = _cli.utils.remove_flags(cmd, ["watch", "w"])
+    header = f"> every {every}s: inductiva {cmd}"
+
+    def action(fout: TextIO = sys.stdout):
+        fout.clear()
+        func(args, fout=fout)
+
+    with ansi_pager.PagedOutput(header, "Press 'q' to close.") as pager:
```

Comment:
There is more to the paginator than simply pressing `q` to exit; you can scroll line by line and page by page.
Hence the default footer.